Make state tooltip nice and tabbable

vkoves opened the pull request:
## Overview

Makes it so the map of states on the homepage uses a Bootstrap tooltip and is now tabbable and shows the tooltip. Here's a little demo, starting with tabbing and then moving to using the mouse:

https://user-images.githubusercontent.com/3187531/167757160-03dd37bb-dd6e-43b4-bffa-7d791ac9c675.mp4

Closes #78

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Try tabbing through the map and confirm you see the tooltips
* Hover over the map and confirm you see reasonable tooltipss

vkoves commented on src/styles/global.css:
```diff
@@ -13,6 +13,10 @@ body, * {
 }
 
+body *:focus {
+    outline: dashed 0.25rem #993404;
```
This makes a consistent global focus behavior using a dashed orange outline, which is more obvious than Chrome's default.

vkoves commented:
I forgot to handle `focusout`, so the tooltip sticks around if you click out after focusing:

![Screenshot from 2022-05-10 21-43-47](https://user-images.githubusercontent.com/3187531/167758391-c54fbd0f-aa93-44fc-9527-000c8c78bf3c.png)

derekeder reviewed (approved):
nice! works as expected

vkoves commented:
@derekeder - the one focus issue is now fixed:

![Peek 2022-05-12 12-42](https://user-images.githubusercontent.com/3187531/168136384-60389bbb-5834-4e87-a4d8-05dbeef55193.gif)

I also moved the `const` function declarations in this file to actual `function` declarations. It's just personal opinion but I find those easier to read since you start each line saying it's a function, so it's just a bit more explicit.
